setup.py

```diff
@@ -1,42 +1,151 @@
+import distutils
 import glob
 import os
+import shlex
 import setuptools
 import setuptools.command.build_ext
+import setuptools.command.develop
+import setuptools.command.install
 import setuptools.dist
 import sys
 import setuptools.command.install
 import cellprofiler.utilities.version
 
+# Recipe needed to get real distutils if virtualenv.
```

Comment:
You’re much nicer than me.

I intentionally dropped anything required to build a CellProfiler executable on OS X from a virtual environment (e.g. OS X requires someone to crate symbolic links from the system site-packages  wxPython to the virtual environment site-packages and create an app bundle for your virtual environment’s Python—it’s horrible, LOL). I felt it’d be easier to keep documentation and our build system updated with possible py2app and py2exe fixes than our setup.py. I also wanted to maintain platform neutrality (see my other comments).